cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,46 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+let user;
+let article;
+
+describe('Conduit', () => {
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
It's not clear what page you are visiting here. Please, specify the URL or add a comment explaining why you are visiting the root page.